.github/workflows/soroban-rpc.yml

```diff
@@ -70,6 +70,12 @@ jobs:
         run: |
           rustup update $PREFLIGHT_RUST_VERSION
       - run: rustup target add ${{ matrix.rust_target }}
+      - if: matrix.os == 'windows-latest'
+        name: Configure preflight library to use explicit rust version
```

Comment:
This step is needed since GHA runner might have a newer version, and we want to pin the desired version.